.github/ISSUE_TEMPLATE/openviz-summit-topic-proposal.md

```diff
@@ -2,14 +2,14 @@
 name: Open Visualization Collab Summit Session Proposal
 about: Submit a proposal for a session at the Open Visualization Collab Summit 
 title: 'Open Visualization Session Proposal: <Project Name> <Session Title>'
-labels: New York 2023, Collaborator Summit, Session Proposal
+labels: OpenViz London 2024, Collaborator Summit, Session Proposal
```

Comment:
Can we use the spelling `OpenVis`?

Reply:
Yes! I used the same abbreviation that was in the file name, happy to fix.